app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -108,13 +116,6 @@ class EditBookmarkFragment : Fragment(), CoroutineScope {
         updateBookmarkFromObservableInput()
     }
 
-    override fun onPause() {
```

Comment:
I moved the code to move bookmarks, as I noticed a occasional race condition by backgrounding it during onPause. If needed, I can separate this fix from the Telemetry PR.